src/core/link-to-dfn.js

```diff
@@ -46,6 +44,20 @@ export function run(conf, doc, cb) {
         }
       }
     });
+    if (listOfDuplicateDfns.length > 0) {
+      var dfnsList = "";
+      for (var i = 0; i < listOfDuplicateDfns.length; i++) {
+        dfnsList += "["+ parseInt(i+1) + "](#" +
+          listOfDuplicateDfns[i].attr("id") + ")" + " ";
+      }
+      pub(
+        "error",
+        "Duplicate definitions of '" +
+          title + "' at: " +
+          dfnsList
+      );
+      listOfDuplicateDfns = [];
```

Comment:
No need for this, let the garbage collector deal with it.

Reply:
On it